admin/controllers/pupil-pin.js

```diff
@@ -116,6 +133,7 @@ const postGeneratePins = async (req, res, next) => {
   }
   let school
   try {
+    await pinGenerationEligibilityService.determinePinGenerationEligibility(isLiveCheck)
```

Comment:
This is essentially a validator isn't it? I wonder if we should call it that?

Reply:
@jon-shipley renamed to `businessAvailabilityService` as agreed